controllers/defaults/config.go

```diff
@@ -25,16 +26,29 @@ const (
 	deprecatedTemplateName = "poison-pill-default-template"
 )
 
-var defaultTemplateRef = &corev1.ObjectReference{
+var DefaultTemplateRef = &corev1.ObjectReference{
 	Kind:       "SelfNodeRemediationTemplate",
 	APIVersion: "self-node-remediation.medik8s.io/v1alpha1",
 	Name:       "self-node-remediation-resource-deletion-template",
 }
 
+var DefaultSelector = metav1.LabelSelector{
+	MatchExpressions: []metav1.LabelSelectorRequirement{
+		{
+			Key:      utils.ControlPlaneRoleLabel,
+			Operator: metav1.LabelSelectorOpDoesNotExist,
+		},
+		{
+			Key:      utils.MasterRoleLabel,
+			Operator: metav1.LabelSelectorOpDoesNotExist,
+		},
+	},
+}
```

Comment:
By using new default selector (`worker role` ->  `!control-plane && !master role`) we should also update it in two other places:

- Sample file - https://github.com/medik8s/node-healthcheck-operator/blob/main/config/samples/remediation_v1alpha1_nodehealthcheck.yaml#L9-L10 
- README -  In the example under `NodeHealthCheck Custom Resource` - https://github.com/medik8s/node-healthcheck-operator/blob/main/docs/README.md#nodehealthcheck-custom-resource

Reply:
Maybe also in the table under `NodeHealthCheck Custom Resource` - https://github.com/medik8s/node-healthcheck-operator/blob/main/docs/README.md#nodehealthcheck-custom-resource for `selector` row.

Reply:
thx, fixed the default CR in the README. The table is about the default values of the fields in the CR, not the overall default CR. And the sample is a sample, not the default CR